tests/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 # Enable testing for current directory and below.
 enable_testing(true)
 
+# TODO
+#add_executable(py_test py_test.cpp)
+#add_dependencies(py_test taraxad)
+#target_compile_definitions(py_test PRIVATE NODE_EXECUTABLE_PATH=\"${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/taraxad\")
+#add_test(py_test ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/py_test)
+
```

Comment:
is this supposed to be commented out ? How is the new test run then ?

Reply:
The python tests are in progress, I believe there's a TODO above those commented lines that suggests that it should be uncommented eventually.
How this test will work: we compile a launcher executable that will itself just run the script `tests/py/run.sh` which does all the job. It seems ugly that we have to create an intermediate executable for this - I made it to stick to the pattern we have here. But I can also implement it via cmake custom target if you want.

Reply:
well plain TODO does not say much to me of what should be eventually done. But if I understand you correctly you are going to finish these python tests and uncomment it before its merged so ok :) Will keep this convo opened then...